src/main/java/fr/moribus/imageonmap/image/ImageRendererExecutor.java

```diff
@@ -68,25 +68,38 @@ static public void render(final URL url, final ImageUtils.ScalingType scaling, f
             @Override
             public ImageMap run() throws Throwable
             {
-                final URLConnection connection = url.openConnection();
+                URL url2=null;
+                if(url.toString().contains("https://imgur.com/")){
+                    if(url.toString().contains("gallery/")){
+                        throw new IOException("We do not support imgur gallery yet, please use direct link to image instead (right click on the picture you want to use then select copy link:) ");
```

Comment:
Extra `:` at the end, and this should be translatable.

```suggestion
                        throw new IOException(I.t("We do not support imgur gallery yet: please use direct link to image instead (right click on the picture you want to use, then select “copy link” or similar)."));
```